cmd/swarmctl/config/inspect.go

```diff
@@ -0,0 +1,54 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/docker/swarmkit/api"
+	"github.com/docker/swarmkit/cmd/swarmctl/common"
+	gogotypes "github.com/gogo/protobuf/types"
+	"github.com/spf13/cobra"
+)
+
+func printConfigSummary(config *api.Config) {
+	w := tabwriter.NewWriter(os.Stdout, 8, 8, 8, ' ', 0)
+	defer w.Flush()
+
+	common.FprintfIfNotEmpty(w, "ID\t: %s\n", config.ID)
+	common.FprintfIfNotEmpty(w, "Name\t: %s\n", config.Spec.Annotations.Name)
+	if len(config.Spec.Annotations.Labels) > 0 {
+		fmt.Fprintln(w, "Labels\t")
+		for k, v := range config.Spec.Annotations.Labels {
+			fmt.Fprintf(w, "  %s\t: %s\n", k, v)
+		}
+	}
+
+	common.FprintfIfNotEmpty(w, "Created\t: %s\n", gogotypes.TimestampString(config.Meta.CreatedAt))
```

Comment:
Non-blocking:  since the config data is not redacted, should that be printed out as well?  (although it may not be ascii, so understandable if the preference is to avoid it).